services/cadip/config/cadip_ws_config.yaml

```diff
@@ -1,17 +1,24 @@
+
```

Comment:
Add configuration to this file: some values can change depending on the environment, so we need to pass them as environment variables